### PR TITLE
Add comment that demangles calls if asm.demangle=true

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -3946,6 +3947,25 @@ static void ds_print_ptr(RDisasmState *ds, int len, int idx) {
 #endif
 }
 
+static void ds_print_demangled(RDisasmState *ds) {
+	if (!ds->show_comments || !ds->asm_demangle) {
+		return;
+	}
+	RCore *core = ds->core;
+	RFlagItem *f = NULL;
```
Comment (radare) [inline libr/core/disasm.c]: no need to initialize, you can assign it below too if you add some braces

File: libr/core/cbin.c [hunk 2]
```diff
@@ -2067,6 +2067,7 @@ static int bin_symbols(RCore *r, int mode, ut64 laddr, int va, ut64 at, const ch
 				RFlagItem *fi = r_flag_set (r->flags, fnp, addr, symbol->size);
 				if (fi) {
 					r_flag_item_set_realname (fi, n);
+					fi->demangled = !!sn.demname;
```
Comment (radare) [inline libr/core/cbin.c]: just a cast should be fine (bool)(size_t)

File: libr/core/disasm.c (hunk 1 above)
Comment (kazarmy) [reply libr/core/disasm.c]: ok f1680776356163b9c30486497b89525a92a744c1

File: libr/core/cbin.c (hunk 2 above)
Comment (kazarmy) [reply libr/core/cbin.c]: ok f1680776356163b9c30486497b89525a92a744c1

**Pull request merged**